release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -4385,4 +4385,29 @@ $ oc adm release info 4.12.46 --pullspecs
 [id="ocp-4-12-46-updating"]
 ==== Updating
 
+To update an existing {product-title} 4.12 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#updating-cluster-cli[Updating a cluster using the CLI].
+
+[id="ocp-4-12-47"]
+=== RHSA-2024:0198 - {product-title} 4.12.47 bug fix and security update
+
+Issued: 2024-01-17
+
+{product-title} release 4.12.47, which includes security updates, is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2024:0198[RHSA-2024:0198] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2024:0200[RHSA-2024:0200] advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.12.47 --pullspecs
+----
+
+[id="ocp-4-12-47-bug-fixes"]
+==== Bug fixes
+
+* Previously, the `spec.storage.deviceClasses.thinPoolConfig.overprovisionRatio` value on a Logical Volume Manager Storage (LVMS) cluster custom resource could only be set to a minimum of 2. With this release, the `spec.storage.deviceClasses.thinPoolConfig.overprovisionRatio` value can now be set to as low as 1, which disables overprovisioning. (link:https://issues.redhat.com/browse/OCPBUGS-24480[*OCPBUGS-24480*])  
```

Comment:
Tiny nit: I think if we're talking about values of specs, we want to put the actual values in code format:
```suggestion
* Previously, the `spec.storage.deviceClasses.thinPoolConfig.overprovisionRatio` value on a Logical Volume Manager Storage (LVMS) cluster custom resource could only be set to a minimum of `2`. With this release, the `spec.storage.deviceClasses.thinPoolConfig.overprovisionRatio` value can now be set to as low as `1`, which disables overprovisioning. (link:https://issues.redhat.com/browse/OCPBUGS-24480[*OCPBUGS-24480*])  
```

Reply:
good catch! done :)